ci/main.go

```diff
@@ -406,7 +395,7 @@ func generateApiID(ctx context.Context, img *dagger.Container) (*dagger.Containe
 	}
 
 	target = "g/queue/v1/queueid.go"
-	queueCmd := append(goCmd, "queue", "Queue", "QueueErr", "q", "queue", "Q", "errqueue")
+	queueCmd := append(goCmd, "-p", "queue", "Queue", "q", "queue")
```

Comment:
Yeah, I removed all the other uses of the id generator.  You can no longer generate the "error" version or the combo of the "id and error" because we are using simpler errors.